samples/v1.0/Elements/Action.OpenUrl.json

```diff
@@ -11,7 +11,7 @@
 	"actions": [
 		{
 			"type": "Action.OpenUrl",
-			"title": "Action.OpenUrl",
+			"title": "long text test, long text test, long text test, long text test, long text test, long text test,long text test",
```

Comment:
I will create a new sample card if we want to have one, otherwise, I will revert this change once the review is over.